src/components/shared/TractionMenuItem.vue

```diff
@@ -1,24 +1,55 @@
 <template>
   <li
     :class="[
-      'pl-2 pr-2 p-0 m-0', // No margins, padded left and right
-      'hover:bg-sp hover:text-white transition-colors duration-200 ease-in-out',
+      'pl-2 pr-2 p-0 m-0 rounded-t-md', // No margins, padded left and right
+      `hover:${getColor} hover:text-white transition-colors duration-200 ease-in-out`,
       'cursor-pointer',
-      active ? 'text-white bg-sp' : 'text-sp',
+      active ? `text-white ${getColor}` : `${getTextColor}`,
     ]"
   >
-    <slot />
+    <div :class="active ? `border-1  border-${getColor}` : ``">
+      <slot />
+    </div>
   </li>
 </template>
 
 <script>
 export default {
+  /**
+   * # TractionMenuItem
+   *
+   * Tailwind component to display a menu-item/tab-item within TractionMenu.
+   * Support for two colors.
+   *
+   * @example
+   * <template>
+   *   <div>
+   *      <traction-menu :border="true">
+   *         <traction-menu-item/>
+   *      </traction-menu>
+   *   </div>
+   * </template>
+   */
   name: 'TractionMenuItem',
   props: {
     active: {
       type: Boolean,
       default: false,
     },
+    color: {
```

Comment:
I think there might be a more Vue/JavaScript way to do this and encapsulate in a single function maybe? Computed properties can have a getter and setter. Maybe we could go through it on Friday.

Reply:
It would be great if you can describe this.

Reply:
It would be great if you can describe this on Friday.

Reply:
I have pushed in new changes

Reply:
Much nicer. Well done.